File: src/main/java/com/hjj/apiserver/repositroy/SavingRepository.java
package com.hjj.apiserver.repositroy;

import com.hjj.apiserver.domain.saving.Saving;
import com.hjj.apiserver.domain.saving.SavingPK;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface SavingRepository extends JpaRepository<Saving, SavingPK>, SavingRepositoryCustom {
}

File: src/main/kotlin/com/hjj/apiserver/repository/saving/SavingRepository.kt
package com.hjj.apiserver.repository.saving

import com.hjj.apiserver.domain.saving.Saving
import com.hjj.apiserver.domain.saving.SavingPK
import org.springframework.data.jpa.repository.JpaRepository

interface SavingRepository: JpaRepository<Saving, SavingPK>, SavingRepositoryCustom {
}